src/routes/u/[playerId].svelte

```diff
@@ -139,6 +139,15 @@
       await fetcher(`/api/user/${player.id}/giveRole/${role}`, { withCredentials: true });
       refreshPlayerData({ forceRevalidate: true, softRefresh: true });
    }
+
+   let isOculusPlayer : boolean;
+   $: isOculusPlayer = !!$playerData?.profilePicture?.endsWith('oculus.png');
```

Comment:
Can't we check if the user is an oculus player via the method on line 183?

Reply:
You mean this one?  Does that magic number mean Oculus player?
```
{#if parseInt(playerId) >= 70000000000000000}
```

Reply:
If the playerid is larger than 70000000000000000, it's a steam player

Reply:
Oh, nice. So I'll change the way I check if it's a Steam or Oculus player.